src/types/box.js

```diff
@@ -58,6 +60,7 @@ BoxAnnotation.defaults = {
   cornerRadius: undefined, // TODO: v2 remove support for cornerRadius
   display: true,
   label: {
+    borderWidth: undefined,
```

Comment:
I think this new options should be documented. FYI I didn't add borderWidth because I thought that the label in box couldn't have any border (and any background color).
And, maybe I'm wrong, it's never set to the context, therefore is not used to draw the box of the label.

Reply:
It's actually a leftover abd should be removed. I used this instead of setting the labelOpts.borderWidth = opts.borderWidth, but found out later it was actually the box border width that is used.

Reply:
:+1: do you want to draw a rectangle around the label anyway?

Reply:
No, I don't have a use case for that right now, but no objection either. I'd ideally like the label to be a sub element, but the label annotation is too fancy for that. Maybe if the callout was a sub element too.. Needs more thought anyway.

Reply:
I was reviewing this PR and and the usage of sub elements for inner labels (box and line). But you're right, also callout could be.